### DISCUSSION TITLE
JSON Schema Conf 2024

### DISCUSSION THREAD

**Discussion by benjagm** [json-schema-org/community#690] [General]
During the past API Days Paris we had the privilege of partnering with [AsyncAPI and OpenAPI with the API Specs and Standards booth ](https://json-schema.org/blog/posts/apidays-paris-2023-recap) and it resulted in a fantastic experience where we realized about how important and beneficial may be for the JSON Schema Community to have the opportunity to gather in person so we added this to our wish list for 2024.

With this context we followed up the conversation with the API Days Team and suggested this idea of a JSON Schema Track and they are as excited as we are about having the first JSON Schema Conference as part of API Days. YEAAAAH!

### Next steps

This is going to be the event of the year for our Community, but it will require a lot of planning and work. The immediate next steps are:

1. **Decide on the name of the conference**.
2. **Decide the day**: December 3rd (1st day of API Days and quiet) vs December 5th (Last day and same day than AsyncAPI and OpenAPI Confs)
3. **Create a working work** to have more people involved in the organization. Join `#temp-jsonschema-conf24` [slack](https://json-schema.org/slack) channel to know more.

### Getting involved

Please add a comment to this issue with your proposal for a name for the conference, your preference with the day and any other thought or piece of feedback you may have.

Please, consider joining the working group to organize the conference. it will be a good amount of work and we'll need extra hands. First step is joining `#temp-jsonschema-conf24` [slack](https://json-schema.org/slack) channel.

**Comment by TamannaVerma99** [comment]
How's JSON Schema Symphony (JSS) 
or JSON Schema Symposium

**Comment by benjagm** [comment]
JSON Schema Conf / #JSC24
